### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by unknown10777** [community/community#16704] [Mobile]
Request user customizable language and translate readme.md using google translation api

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.64.0 (66902451)
Device: iPhone SE 2 (iOS 14.0)
</details>

**Comment by mxie** [comment]
Hi @unknown10777  - thanks for your feedback but can you tell me more about what you mean by "request user customizable language"?

**Reply by unknown10777** [reply]
> Hi @unknown10777  - thanks for your feedback but can you tell me more about what you mean by "request user customizable language"?

Like telegram's `tg://setlanguage/`

**Reply by unknown10777** [reply]
Like the telegram custom language function.
